projects/ngrx.io/content/guide/signals/signal-store/testing.md

```diff
@@ -0,0 +1,54 @@
+# Testing
+
+There are two scenarios when it comes to testing:
+
+1. Testing the `signalStore` itself,
+2. Testing a component/service which uses the Signal Store.
+
+Whereas in the first scenario, mocking will happen for dependencies of the Signal Store, scenario 2 might require to mock the Signal Store itself.
+
+The examples are using the Jest API, but the same principles apply to other testing frameworks.
+
+## Testing the Signal Store
+
+### Testing without `TestBed`
+
+The Signal Store is a function that returns a class. That means a test can instantiate the class and test it without the TestBed:
+
```

Comment:
I initially suggested providing both guides - with and without `TestBed`. After reconsideration, I think it would be enough to have only the guide with `TestBed` because:

- Without `TestBed` we cannot test the `onDestroy` hook.
- SignalStore can be tested without TestBed only when there are no injected services, reactive methods, and/or effects which is a rare case.

Reply:
Yeah, I will shorten it, but I think it is important to explain why our guide contains only the `TestBed` scenario (you mentioned some reasons already)

Reply:
Yes, it can be mentioned. E.g:

```md
## Testing a SignalStore

Guide with `TestBed` ...

<div class="alert is-helpful">

SignalStore can also be tested without `TestBed`...

</div>
```